### PR TITLE
Update Canvas to 35mm Camera

### PR DESCRIPTION
**Pull request by jtmcginnis**
Add rocket shoes costume

### CODE REVIEW

File: main.lua [hunk 1]
```diff
@@ -46,7 +46,7 @@ local function start()
         METEOR_SHOT = AlphaAPI.createFlag(),
         ENTROPY_TEAR = AlphaAPI.createFlag(),
         PAINTED = AlphaAPI.createFlag(),
-        CRYSTAL = AlphaAPI.createFlag()
```
Comment (Meowlala) [inline main.lua]: why was a comma added here?

File: main.lua [hunk 2]
```diff
@@ -187,7 +188,7 @@ local function start()
     -- Use Callbacks
     -------------------
     mod:AddCallback(ModCallbacks.MC_USE_ITEM, Alphabirth.triggerPuzzlebox, ITEMS.ACTIVE.PUZZLE_BOX.id)
-    mod:AddCallback(ModCallbacks.MC_USE_ITEM, Alphabirth.onUseCanvas, ITEMS.ACTIVE.CANVAS.id)
```
Comment (Meowlala) [inline main.lua]: these should probably be switched to
ITEMS.ACTIVE.CANVAS:addCallback(AlphaAPI.Callbacks.ITEM_USE, Alphabirth.onUseCanvas)
etc

File: main.lua [hunk 3]
```diff
@@ -231,31 +232,59 @@ end
 ---- ACTIVE ITEM LOGIC
 -------------------------------------------------------------------------------
 -------------------
--- Canvas
+-- CAMERA
 -------------------
-function Alphabirth.onPickupCanvas()
-    if not alpha_mod.data.run.canvas_type then
-        alpha_mod.data.run.canvas_type = nil
+local function findClosestEnemy(entity)
```
Comment (Meowlala) [inline main.lua]: AlphaAPI has a findNearestEntity function, what is lacking?

Comment (jtmcginnis) [inline main.lua]: I was getting grid entities, decorations, etc.. with that function

**Pull request merged**